sdcm/nemesis.py

```diff
@@ -4860,6 +4863,11 @@ def finalizer(exc_type, *_):
                 datacenters = list(self.tester.db_cluster.get_nodetool_status().keys())
                 self._write_read_data_to_multi_dc_keyspace(datacenters)
 
+                for keyspace in system_keyspaces + ["keyspace_new_dc"]:
+                    strategy = ReplicationStrategy.get(node, keyspace)
+                    strategy.replication_factors_per_dc.update({new_dc_name: 0})
```

Comment:
@aleksbykov @timtimb0t according to @soyacz comment and provided [link](https://enterprise.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/decommissioning-data-center.html) to doc
can we just pop the key 

```
strategy.replication_factors_per_dc.pop(new_dc_name)
```

Reply:
according to my comment, and logs provided, we already pop this by `temporary_replication_strategy_setter` upon exit. Just something changed that we need to wait some time (I don't know why, one idea is schema agreement, but it's just my guess).